src/operator/crop.cc

```diff
@@ -21,9 +21,15 @@ Operator* CropProp::CreateOperator(Context ctx) const {
 DMLC_REGISTER_PARAMETER(CropParam);
 
 MXNET_REGISTER_OP_PROPERTY(Crop, CropProp)
-.describe("Crop the 2nd and 3rd dim of input data, with the corresponding size of h_w or "
-"with width and height of the second input symbol, i.e., with one input, we need h_w to "
-"specify the crop height and width, otherwise the second input symbol's size will be used")
+.describe(R"code(
+
+.. note:: `Crop` is deprecated use `slice`.
```

Comment:
do we want to keep this `Crop` ? I think it will be pretty confusing for the user.